mesonbuild/mesonlib.py

```diff
@@ -183,6 +184,20 @@ def __hash__(self):
     def relative_name(self):
         return os.path.join(self.subdir, self.fname)
 
+def source_strings_to_files(source_dir, subdir, sources):
+    results = []
+    for s in listify(sources):
+        if hasattr(s, 'held_object') or isinstance(s, File):
+            pass
+        elif isinstance(s, str):
+           # self.validate_within_subproject(self.subdir, s)
```

Comment:
__[Flake8]__

indentation is not a multiple of four (comment)


<!-- Comment by [SideCI](https://sideci.com) -->